### ISSUE TITLE
Working with cloudwatch

### ISSUE THREAD

**Issue by gzholder** [9apps/elasticsearch-cloudwatch#7]
Will this pull FROM cloudwatch to my ES cluster or TO cloudwatch from my ES cluster?
I'm trying to grab the logs in cloudwatch logs and send them over to my ES cluster.

**Comment by locomundo** [comment]
No, this sends ES statistics to CloudWatch, so FROM ElasticSearch TO CloudWatch.
